pkg/loop/internal/client.go

```diff
@@ -162,3 +163,25 @@ func isErrTerminal(err error) bool {
 	}
 	return false
 }
+
+func wrapRpcErr(err error) error {
```

Comment:
This function is net-new.  Need it to allow the clients to pass tests that verify errors act the same regardless of running locally or in LOOP.